pysatModels/tests/test_utils_extract.py

```diff
@@ -477,3 +481,102 @@ def test_alternate_output_names(self):
         assert np.all(alt_diff <= 1.0)
 
         return
+
+
+@pytest.mark.skipif(pysat_version_major < 3 or (pysat_version_major == 3
+                                                and (pysat_version_minor < 1)),
```

Comment:
`packaging.version` could be used to simplify.  Technically a new requirement, but in the downstream dependencies anyway.  See pysat/pysat#965

Reply:
`packaging.version.Version(pysat.__version__) < packaging.version.Version('3.1.0')` yields a bool

Reply:
I like the simplicity, and if it isn't truly a new dependency, I am onboard.